packages/@okta/vuepress-site/docs/concepts/okta-data-model/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Okta Data Model
+---
+
+# Okta Data Model
+
+This page provides a high-level overview of the Okta data model.
+
+Entities inside Okta are referred to as resources. Each Okta resource contains:
+
++ Attributes, such as `name` or `created` timestamp.
++ Links that describe relationships or actions that you can take on the resource. For example, a `self` link on a user would reference itself, and a `deactivate` link would move the lifecycle state of the user.
++ A Profile that allows you to store information about the object. For example, a User profile has an `email` attribute that contains the value for the email address.
+
+When you `sign up` for Okta, a private data space is created for you that is represented as an `organization` in Okta. As an Okta customer, you own your organization and everything in it: Applications, Directories, Users, Groups, Policies, and so on.
+
+A top-level `Directory` resource is available for your user base in your organization. Directories can either be hosted inside Okta or can be used to replicate outside user directories, as in the case of Active Directory or LDAP.
+
+Your end users are modeled inside Okta as [`Users`](/docs/reference/api/users/). Every User is unique within a Directory, with this uniqueness tied to an email address and username. Users have a schema that is defined to control what values are stored on the user's profile.
```

Comment:
Uniqueness for a user is determined by their Okta user ID. The username must be unique, but it _can_ be changed (and often is when, for example, someone has their name changed) but the email address does _not_ need to be unique and often isn't.

Reply:
@joelfranusic-okta How is this instead?
Your end users are modeled inside Okta as [`Users`](/docs/reference/api/users/). Users are associated with a email address and a username. The username must be unique, but it can be changed (and often is when, for example, someone has their name changed). Abd the email address doesn't need to be unique and often isn't. Uniqueness for a user is determined by their Okta user ID.

Reply:
PERFECT